tests/integration/test_startup_scripts/configs/config.xml

```diff
@@ -0,0 +1,14 @@
+<clickhouse>
+    <startup_scripts>
```

Comment:
Is `prepare_system_log_tables_on_startup` added to make it possible to use system tables in startup scripts? We can also enable the setting and do something with system table in test then